src/webserver/impl/WhitelistHandler.h

```diff
@@ -27,7 +27,7 @@
 #include <unordered_map>
 #include <unordered_set>
 
-namespace web::detail {
+namespace web {
```

Comment:
I think if you move it out of the detail namespace you probably want to move the file out of impl too?